src/pages/blog-post.jsx

```diff
@@ -46,6 +60,23 @@ class BlogPost extends React.Component {
     }
   }
 
+  async componentDidUpdate(prevProps) {
```

Comment:
Here too, use Hooks instead of life-cycle methods

Reply:
> Here too, use Hooks instead of life-cycle methods

hey surabhi we take this up in the enhancement program for the starter apps